### ISSUE TITLE
Meetup 5

### ISSUE THREAD

**Issue by adaptive** [madeiratech/talks#2]
Check Meetup 5 info is accurate and sponsors.
https://github.com/madeiratech/talks/blob/master/2017/5.md

**Comment by dethos** [comment]
Yes, seems accurate. Not sure if we had a sponsor for food and drinks.

**Comment by dethos** [comment]
According to this page: https://www.meetup.com/Madeira-Tech/events/242058878/

Frontkom sponsored the snacks.

**Comment by adaptive** [comment]
PR #7

**Comment by adaptive** [comment]
@b2rsp please review your meetup
